libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html

```diff
@@ -71,6 +71,7 @@
                 type="text"
                 role="combobox"
                 autocomplete="off"
+                [aria-describedby]="status"
```

Comment:
this just reads the status itself how it is (e.x. `warning`) it doesn't say that this info is about a control state (e.x. Value State Warning)

Reply:
also `[aria-describedby]` -> `[attr.aria-describedby]`